src/mpid/ch4/netmod/ucx/ucx_types.h

```diff
@@ -23,8 +23,8 @@
 /* Active Message Stuff */
 #define MPIDI_UCX_NUM_AM_BUFFERS       (64)
 #define MPIDI_UCX_MAX_AM_EAGER_SZ      (16*1024)
-#define MPIDI_UCX_TAG_USABLE_BITS      (MPIR_TAG_USABLE_BITS >> 1)
-#define MPIDI_UCX_AM_TAG               (MPIDI_UCX_TAG_USABLE_BITS + 1)
+#define MPIDI_UCX_TAG_BITS             (MPIR_TAG_BITS_DEFAULT)
+#define MPIDI_UCX_AM_TAG               (MPIR_TAG_USABLE_BITS + 1)
```

Comment:
@raffenet for consistency with `ch4/ofi` I have removed `MPIDI_UCX_TAG_USABLE_BITS`, which is now replaced by `MPIR_TAB_USABLE_BITS` and defined `MPDI_UCX_TAB_BITS`, to which `tag_bits` is set to in the UCX init function.